tools/api-markdown-documenter/src/test/LoadModel.test.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as Path from "node:path";
+import { fileURLToPath } from "node:url";
+
+import { expect } from "chai";
+
+import { loadModel } from "../LoadModel.js";
+
+const dirname = Path.dirname(fileURLToPath(import.meta.url));
+const testModelsDirectoryPath = Path.resolve(dirname, "..", "..", "src", "test", "test-data");
+
+describe("loadModel", () => {
+	it("Model directory with a single API report", async () => {
+		const modelDirectoryPath = Path.resolve(testModelsDirectoryPath, "simple-suite-test");
+
+		try {
+			await loadModel({ modelDirectoryPath });
+		} catch (error: unknown) {
+			expect.fail(
+				`Expected no error to be thrown, but one was: "${(error as Error)?.message}".`,
+			);
+		}
```

Comment:
Nit: I think the try/catch is redundant in this case, if anything I'd go for a comment above the call that says "We're validating this doesn't throw", but no strong feelings about it.

Reply:
Yeah, that's fair. I'll update it.